compiler/src/iree/compiler/Codegen/SPIRV/test/illegal_configuration.mlir

```diff
@@ -153,43 +153,6 @@ hal.executable private @matmul_tensors {
 
 // -----
 
-#compilation = #iree_codegen.compilation_info<
-    lowering_config  = <tile_sizes = [[32, 64], [4, 2], [0, 0, 4]]>,
-    translation_info = <SPIRVMatmulPromoteVectorize>,
```

Comment:
This test is unrelated to cooperative matrix pipeline?

Reply:
oh, this shouldn't have been dropped. I added it back.